cachix-api/src/Cachix/Api.hs

```diff
@@ -69,6 +69,12 @@ data BinaryCacheAPI route = BinaryCacheAPI
       CachixAuth :>
       Capture "narinfo" NarInfoC :>
       Head
+  , narinfoBulk :: route :-
+      CachixAuth :>
+      "narinfo" :>
+      Summary "Given a list of store hashes, return a list of those that are missing" :>
+      ReqBody '[JSON] [Text] :>
+      Get '[JSON] [Text]
```

Comment:
Although `ReqBody` + `Get` seems very sensible from a REST view, the real world may disagree.
I'd prefer to configure an HTTP cache to cache a POST properly over having to debug one because we're using a corner case.

Reply:
Not sure what do you mean with `having to debug`, is that due to HTTP GET caching? All cachix APIs are not cached atm so there shouldn't be any issues.

Reply:
I was trying to say that GET requests with request bodies are not really standard. They are allowed by the spec but [apparently](https://stackoverflow.com/questions/978061/http-get-with-request-body/983458#983458) they shouldn't influence the response, making them useless.
So yeah we may run into caching issues if we add it ourselves, or if someone decided to proxy it locally.

Reply:
There's an update edit at the bottom. From RFCs published in 2014 body restriction was removed.

However, there's going to be very little caching involved here so better save than sorry.